.changeset/selfish-planets-repeat.md

```diff
@@ -0,0 +1,7 @@
+---
+"@medusajs/workflows": patch
+"@medusajs/medusa": patch
+"@medusajs/utils": patch
```

Comment:
todo: Change to minor, as we are now breaking the feature flags

Reply:
for all 3, yes?